### ISSUE TITLE
When copy pasting text from Word, it is pasted as image, not text

### ISSUE THREAD

**Issue by fr-olivier** [nextcloud/text#2923]
Hello,

Here's the issue:
When copy-pasting some text from Microsoft Word to a text file, it is pasted as an image, not as text itself.

Is there any workaround?
Thank you

**Comment by vinicius73** [comment]
Could you use the issue template with more information about this bug @fr-olivier ?

**Comment by fr-olivier** [comment]
> I think the problem is if your text contain images or drawings they are added to the clipboard, e.g. copying from excel results in a clipboard entry including: `text/html,text/rtf,text/plain,Files` Same if you copy from word.
> 
> In this case the `Files` are a screenshot of the copied part.

No, because even when copy/pasting a few words from Word, it is pasted as a screenshot.

**Comment by juliushaertl** [comment]
Seems like what @susnux described that word is sharing multiple formats and the image paste handling takes precedence there while it shouldn't.

**Comment by mejo-** [comment]
Just stumbled upon https://github.com/Intevation/tiptap-extension-office-paste. No idea if it fixes the problem, but code looks promising to fix some issues when pasting from word at least.

**Comment by juliusknorr** [comment]
https://evercoder.github.io/clipboard-inspector/ may also be helpful for anyone looking further into this

**Comment by mejo-** [comment]
This finally got fixed with https://github.com/nextcloud/text/pull/8897.
